ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly-4.16-upgrade-from-stable-4.15.yaml

```diff
@@ -238,6 +246,29 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test-hypershift-full
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-hypershift-mce-guest
+- as: aws-ipi-ovn-rosa-hcp-upgrade-cp-stage-f7
+  cron: 21 12 1,10,17,24 * *
+  steps:
+    cluster_profile: aws-sd-qe
+    env:
+      AVAILABLE_UPGRADE: "yes"
+      CHANNEL_GROUP: stable
+      COMPUTE_MACHINE_TYPE: m5.2xlarge
```

Comment:
Is the smaller instance type, eg. `m5.xlarge`, match your testing?

Reply:
yes, thanks